DESCRIPTION

```diff
@@ -20,7 +20,8 @@ Authors@R: c(
       comment = c(ORCID = "0000-0002-6714-8611")
     ),
     person("Vincent", "Fulco", role = "ctb"),
-    person("Alec", "Wong", role = "ctb")
+    person("Alec", "Wong", role = "ctb"),
+    person("Alex", "Brodersen", role = "ctb")
```

Comment:
Not related to this line, but it's in the neighborhood. 

Just a heads up that just before merging the PR, we will manually bump the version based on where we will be at the time. No action needed by you, just so you will not be surprised.